build/grafana.yaml

```diff
@@ -37,6 +37,6 @@ datasources:
     datasources:
     - name: Prometheus
       type: prometheus
-      url: http://prom-prometheus-server.metrics.svc.cluster.local:80/
+      url: http://prometheus-server.metrics.svc.cluster.local:80/
```

Comment:
This is the effect of changing the name of the installation from prom to prometheus.

If there is no reason, I think the same as the chart name is good.
It is because the name can be shortened.

In the case of grafana, it's the same as the chart name.

Reply:
I'm a bit hesitant to change this name, because of backward compatibility.

If people are pulling this file down from release branches as part of their install process, this could break things for them, no?

Reply:
That's certainly going to be a problem in that case.
I'll change it to the original release name.

Reply:
Thanks for your comment. Fixed.